sys/freebsd/socket_inet.txt

```diff
@@ -12,8 +12,8 @@ resource sock_in[sock]
 resource ifindex[int32]
 
 sockaddr_in {
+	len	const[16, int8]
```

Comment:
Why did you hard-code the parent structure size here?

Reply:
I tried to use
```
len = len[parent, int8]
```
but this resulted in using `0xffffffff` instead of 16. Not sure why...

Reply:
Did you get 0xffffffff once among 16? Or all the time?
Episodic 0xffffffff is fine, this is a fuzzer in the end. But if you are getting it all the time, it would mean a very serious bug because we rely on len a lot.

Reply:
Either way, the description should say len[parent] if that's what's required. And we need to fix len/figure out what happens.

Reply:
> Either way, the description should say len[parent] if that's what's required. And we need to fix len/figure out what happens.

Using len[]: https://github.com/google/syzkaller/pull/1248/commits/640c44353dc3be29bad668dc0e81e557e737881f

Will retest and report if it works. When I looked earlier today, I saw only 0xffffffff...

Reply:
Retested and the current version using `len[parent, int8]` does what it is supposed to do. So it looks good to me.